src/eCRC_qaautomation/.env.template

```diff
@@ -22,6 +22,5 @@ ORGVERIFICATION_URL=<URL>
 BCSERVIES_CARD_SERIAL_NUMBER=<serial N=number>
 BCSERVIES_CARD_PASSWORD=<password>
 BCSC_CONSENT_URL=<URL>
-USERCONFIRMATION_URL=<URL>
 APPLICATIONFORM_URL=<URL>
 INFORMATIONREVIEW_URL=<URL>
```

Comment:
new line?

Reply:
you mean at the end of the file? It wasn't there before and this is suresh stuff (but i can add it in for consistency 😄 )